Sources/HipMobileUI/ViewModels/Pages/AppetizerViewModel.cs

```diff
@@ -38,23 +38,22 @@ public class AppetizerPageViewModel : NavigationViewModel, IDownloadableListItem
         private ICommand nextViewCommand;
         private bool exhibitUnblocked = true;   // see comment below: remove 'true' if you use the commented out line again
 
-        public AppetizerPageViewModel(string exhibitId) : this(ExhibitManager.GetExhibit(exhibitId)) { }
+        public AppetizerViewModel(string exhibitId) : this(ExhibitManager.GetExhibit(exhibitId)) { }
 
-        public AppetizerPageViewModel(Exhibit exhibit)
+        public AppetizerViewModel(Exhibit exhibit)
         {
             Exhibit = exhibit;
             //exhibitUnblocked = exhibit.Unlocked;      // currently commented out for testing -> no location check required to access exhibit details
```

Comment:
I know you didn't change this, but should it remain commented out? If yes, maybe there should at least be a `// TODO Comment this in again` so it's easily searchable, or even an issue for it on JIRA. Alternatively, the boolean could be set depending on whether this is debug or release mode (see [guide](https://docs.microsoft.com/de-de/dotnet/csharp/language-reference/preprocessor-directives/preprocessor-if)).

Reply:
Good Idea, I will put DEBUG Directive